client/src/service-worker.js

```diff
@@ -1,12 +1,27 @@
 /* eslint-disable no-restricted-globals */
 
+// Import Firebase scripts
+import firebase from "firebase";
+
+self.importScripts("https://www.gstatic.com/firebasejs/8.2.9/firebase-app.js");
+self.importScripts("https://www.gstatic.com/firebasejs/8.2.9/firebase-messaging.js");
```

Comment:
Is this possible to have these file in our bundle instead of downloading them from a third-party server?